cfme/tests/cloud_infra_common/test_retirement.py

```diff
@@ -45,32 +77,38 @@ def retire_vm(small_template, provider):
         small_template: small template fixture, template on provider
         provider: provider crud object from fixture
     """
-    collection = provider.appliance.provider_based_collection(provider)
-    vm = collection.instantiate(random_vm_name('retire'),
-                                provider,
-                                template_name=small_template.name)
-    vm.create_on_provider(find_in_cfme=True, allow_skip="default", timeout=1200)
+    vm = create_vms(small_template.name, provider)[0]
     yield vm
     vm.cleanup_on_provider()
 
 
+@pytest.fixture(scope="function")
+def retire_vm_pair(small_template, provider):
+    """Fixture for creating a pair of generic vms/instances
+
+    Args:
+        small_template: small template fixture, template on provider
+        provider: provider crud object from fixture
+    """
+    vms = create_vms(small_template.name, provider, 2)
+    yield vms
+    for num in range(2):
+        vms[num].cleanup_on_provider()
+
+
 @pytest.fixture(scope="function")
 def retire_ec2_s3_vm(provider):
     """Fixture for creating an S3 backed paravirtual instance, template is a public ec2 AMI
 
     Args:
         provider: provider crud object from fixture
     """
-    collection = provider.appliance.provider_based_collection(provider)
-    vm = collection.instantiate(random_vm_name('retire'),
-                                provider,
-                                template_name='amzn-ami-pv-2015.03.rc-1.x86_64-s3')
-    vm.create_on_provider(find_in_cfme=True, allow_skip="default", timeout=1200)
+    vm = create_vms('amzn-ami-pv-2015.03.rc-1.x86_64-s3', provider, instance_type='m1.small')[0]
```

Comment:
Is it possible for this template name and instance type to be fetched from yamls?